Source/ASNetworkImageNode.h

```diff
@@ -145,6 +145,18 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol ASNetworkImageNodeDelegate <NSObject>
 @optional
 
+/**
+ * Notification that the image node finished downloading an image, with additional info.
```

Comment:
...downloading an image container, ...

Add another line below this for:
* Use this method if your app handles custom <ASImageContainerProtocol> types, such as for animated images.